index.html

```diff
@@ -829,246 +829,192 @@ <h4>
-                  <td>10.5pt</td>
-                </tr>
-                <tr>
-                  <td>六號</td>
-                  <td>7.875pt</td>
-                </tr>
-                <tr>
-                  <td>七號</td>
-                  <td>5.25pt</td>
-                </tr>
-              </tbody>
-            </table>
-
-            <table class="charsize" id="newsize-zhhant" its-locale-filter-list="zh-hant" lang="zh-hant">
-              <thead>
-                <tr>
-                  <th>新印刷字號</th>
-                  <th>點數</th>
-                </tr>
-              </thead>
-              <tbody>
-                <tr>
-                  <td>新五號四行</td>
-                  <td>36pt</td>
-                </tr>
-                <tr>
-                  <td>新一號</td>
-                  <td>24pt</td>
-                </tr>
-                <tr>
-                  <td>新二號</td>
-                  <td>18pt</td>
-                </tr>
-                <tr>
-                  <td>新三號字</td>
-                  <td>16pt</td>
-                </tr>
-                <tr>
-                  <td>新四號</td>
-                  <td>12pt</td>
-                </tr>
-                <tr>
-                  <td>新五號</td>
-                  <td>9pt</td>
-                </tr>
-              </tbody>
-            </table>
-
-            <p its-locale-filter-list="en" lang="en">Size 5 is usually used for text content. Newspapers and magazines use both Size 5 and New Size 5. The acceptable minimum size for the text in content is Size 6  (7.875pt≒2.8mm). If a smaller size is used, it will be difficult to read due to the complex structure of Chinese characters.</p>
-            <p its-locale-filter-list="zh-hans" lang="zh-hans">一般内文主要使用五号字（10.5pt≒3.7mm），而报纸、杂志则使用新五号字（9pt≒3.2mm），两种皆常用。而一般内文字最小使用到六号字（7.875pt≒2.8mm），若小于此尺寸，由于汉字结构复杂，较难阅读。</p>
-            <p its-locale-filter-list="zh-hant" lang="zh-hant">一般內文主要使用五號字（10.5pt≒3.7mm），而報紙、雜誌則使用新五號字（9pt≒3.2mm），兩種皆常用。而一般內文字最小使用到六號字（7.875pt≒2.8mm），若小於此尺寸，由於漢字結構複雜，較難閱讀。</p>
-           </li>
+          <p its-locale-filter-list="en" lang="en">There have been different size systems for Chinese characters. The size system in traditional metal type utilized hào (literally No. ) units, while in the phototypesetting era, Q was used as the sizing unit instead. When it came to desktop publishing, font sizes were determined by the DTP point system which was built into the software itself. Currently, the traditional hào-system is still used for typesetting in many Chinese publications.</p>
+          <p its-locale-filter-list="en" lang="en">These hào-system were not standardized by the various foundries in the past. In addition, point-systems were also different in Anglo-American, Europe Continental, DTP and so on, which resulted in numerous conversion methods between hào-system and point-system. The following table lists their most common corresponding conversions as a reference. It is not normative information.</p>
```

Comment:
"These hào-system were" needs an 's' after system (assuming that plural is meant)

Reply:
"in Anglo-American, Europe Continental, DTP, and"
Is the meaning "in Anglo-American systems, Europe Continental systems, DTP and", or "in Anglo-American DTP, Europe Continental DTP, and"
I suggest an Oxford comma after DTP in the first case.

Reply:
Actually I was meaning "in Anglo-American point systems, Europe Continental point systems, DTP point systems and other systems"

Yes, and Oxford comma is better!

Reply:
Edited.